DOCS-3197: Update support article and linked snippets (#1300)
truenas/documentation: content/CORE/System/Support.md

Users are also welcome to report bugs, vote for, or suggest new TrueNAS features in the project Jira instance.

Customers who purchase iXystems hardware or that want additional support must have a support contract to use iXystems Support Services. 

### TrueNAS Community

The [TrueNAS Community](https://www.truenas.com/community/) is an active online resource for asking questions, troubleshooting issues, and sharing information with other TrueNAS users.
[Registration](https://www.truenas.com/community/register/) is required for posting.

> New users are encouraged to privide a brief [introduction](https://www.truenas.com/community/forums/introductions.25/) of themselves and to review the [forum rules](https://www.truenas.com/community/threads/forum-rules.45124/) before posting.

Reviewer: Typo
```suggestion
New users are encouraged to provide a brief [introduction](https://www.truenas.com/community/forums/introductions.25/) of themselves and to review the [forum rules](https://www.truenas.com/community/threads/forum-rules.45124/) before posting.
```

If you encounter a bug or other issue while using TrueNAS, create a bug report in the [TrueNAS Jira Project](https://jira.ixsystems.com/projects/NAS/).
The web interface provides a form to report issues without having to log out.
It is recommended to search the project first to see if the issue is already reported.
You need to [create a Jira account](https://jira.ixsystems.com/secure/Signup!default.jspa) before creating a bug ticket.

To report an issue using the web interface, go to **System > Support**.

![UIBugReport](/images/CORE/12.0/UIBugReport.png "Writing a Bug Report")

Enter your Jira information in the **Username** and **Password** fields to verify your account credentials and activate the **Submit** button.
The **Category** dropdown list has a large number of options.
Choose the category that best fits where you encountered the issue.

> Attach a debug file and any screenshot to your bug ticket is helps speed up the response and resolution to find and fix the bug.

Reviewer: Extraneous word
```suggestion
Attach a debug file and any screenshot to your bug ticket helps speed up the response and resolution to find and fix the bug.
```

Debug files are always attached to the ticket privately and are deleted when the ticket is resolved.

Keep the information in **Subject** brief and informative.

> Providing a short, descriptive subject allows the community to better find and respond to your issue.

Reviewer: ```suggestion
Provide a short, descriptive subject to help the community find and respond to your issue.
```

Be sure to add valid email addresses and phone numbers for the contacts to be quickly notified of any issues.

> You can also toggle on automatic iXsystesms support alerts in the system console menu (`/etc/netcli` in the **Shell**).

Reviewer: ```suggestion
You can also enable automatic iXsystesms support alerts in the system console menu (`/etc/netcli` in the **Shell**).
```

> You must disable failover in TrueNAS High Availability systems before toggling on this option.

Reviewer: ```suggestion
Failover on TrueNAS High Availability systems must be disabled before activating automatic alerts.
```
